rs/crypto/ecdsa_secp256r1/src/lib.rs

```diff
@@ -374,7 +554,7 @@ impl PublicKey {
     /// valid signature, then (r,-s) is also valid. To avoid this malleability,
     /// some systems require that s be "normalized" to the smallest value.
     ///
-    /// This normalization is quite common on secp256k1, but is virtually
+    /// This normalization is quite common on secp256r1, but is virtually
```

Comment:
Was this change intentional? The text now says: "normalization is common on r1, but is unknown for r1".

Reply:
Oops search and replace gone bad